app/routes/organen.js

```diff
@@ -22,10 +19,16 @@ export default class OrganenRoute extends Route {
       bestuurseenheid.get('id')
     );
 
+    const gemeenteBestuursorganen = [];
     const decretaleBestuursorganen = [];
     const nietDecretaleBestuursorganen = [];
     await Promise.all(
       bestuursorganen.map(async (orgaan) => {
+        const isGemeentelijk = await orgaan.isGemeentelijk;
+        if (isGemeentelijk) {
+          gemeenteBestuursorganen.push(orgaan);
```

Comment:
I feel that this is starting to get a bit convoluted, but I don't immediately have a better solution